src/components/FileRepo/FileRepo.js

```diff
@@ -107,9 +107,30 @@ const ArrangerContainer = styled(Row)`
     border-right: solid 1px #e0e1e6;
   }
 
+  & .tableToolbar {
+    color: ${({ theme }) => theme.greyScale9};
+    & .group .dropDownButtonContent {
+      color: ${({ theme }) => theme.greyScale9};
+    }
+    & .group button {
+      color: ${({ theme }) => theme.greyScale9};
+    }
+  }
+
+  .pagination-bottom .-pagination {
+    color: ${({ theme }) => theme.greyScale9};
+    select {
+      color: ${({ theme }) => theme.greyScale9};
+    }
+  }
+
   div.sqon-view {
     flex-grow: 1;
   }
+
+  & .sqon-clear.sqon-bubble {
+    font-weight: bold;
+  }
```

Comment:
this is starting to look like `beagle.css` territory, we may want to move beagle into KF-portal shortly